api/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=mongodb://localhost/db
```

Comment:
Instead of having a .env file we should have a .env-sample file which is a reference to .env file, the .env file is included in the .gitignore file and should not be pushed to the repository bc there is sensitive data inside.